src/handlers.go

```
@@ -39,60 +45,114 @@ type githubGitignoreClient struct {
 }
 
 func InitHandlers() {
-	gocmd.HandleFlag(fncInit, func(cmd *gocmd.Cmd, args []string) error {
-		language, location, err := handleParams(cmd.FlagArgs(fncInit)[1:])
+	gocmd.HandleFlag(fncList, func(cmd *gocmd.Cmd, args []string) error {
```

golangci-lint (2.11.4)

[error] Error return value of `gocmd.HandleFlag` is not checked

(errcheck)